pages/common/ag.md

```diff
@@ -25,3 +25,7 @@
 - Find files with a name matching "foo":
 
 `ag -g {{foo}}`
+
+- Find files containing "foo" on directory bar/baz:
+
+`ag foo {{bar/baz}}`
```

Comment:
Add token syntax around `foo`.

Also, is `baz` a subdirectory in `bar`? I feel like using the standard path syntax may be more readable and consistent.
```suggestion
`ag {{foo}} {{path/to/directory}}`
```